TrackingTools/Records/interface/TfGraphRecord.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TrackingTools_Records_TfGraphRecord_TfGraphRecord_h
+#define TrackingTools_Records_TfGraphRecord_TfGraphRecord_h
```

Comment:
The CMS convention for the header guard would be
`TrackingTools_Records_TfGraphRecord_h`